.nvmrc

```diff
@@ -1 +1 @@
-v8.14.1
+8
```

Comment:
I have a hunch that this is a problem with windows and line endings as per [this comment](https://github.com/creationix/nvm/issues/1540#issuecomment-446258378).

We should add the trailing line not because it works fine for everyone else but because the [docs](https://github.com/creationix/nvm#nvmrc) state:

`The contents of a .nvmrc file must be the <version> (as described by nvm --help) followed by a newline. No trailing spaces are allowed, and the trailing newline is required.`

Reply:
That makes so much more sense, thanks @SeanPrashad